numba_dpex/core/kernel_interface/dispatcher.py

```diff
@@ -441,6 +445,56 @@ def _determine_kernel_launch_queue(self, args, argtypes):
             else:
                 raise ExecutionQueueInferenceError(self.kernel_name)
 
+    def _raise_invalid_kernel_enqueue_args(self):
+        error_message = (
+            "Incorrect number of arguments for enqueuing numba_dpex.kernel. "
+            "Usage: device_env, global size, local size. "
```

Comment:
The error message is wrong and seems to have been not updated for the last two years even as we made many API changes!! Please do not use this old error message.

Reply:
Fixed.